Source/Core/DiscIO/VolumeWad.h

```diff
@@ -10,6 +10,7 @@
 #include <vector>
 
 #include "Common/CommonTypes.h"
+#include "DiscIO/Blob.h"
```

Comment:
Either remove the BlobType forward declaration, or remove this include and add an IBlobReader forward declaration.